src/components/Layout/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Header from "./Header";
+import Footer from "./Footer";
+import { mainContainer } from "./layout.css";
+
+interface ILayoutProps{
+    children: React.ReactNode
+}
+
+export default function Layout({children}:ILayoutProps){
```

Comment:
children을 직접 선언하는 방법이 있고 PropsWithChildren을 사용하는 방법이 있어서 취향껏 사용해주세요

```suggestion
export default function Layout({children}:PropsWithChildren<ILayoutProps>){
}
```